### ISSUE TITLE
Examples appear broken

### ISSUE THREAD

**Issue by callumgare** [ankitpokhrel/tus-php#284]
Either the examples are all broken or I'm doing something very stupid. When I try to upload anything using the example client website it fails for what looks like CORS related reasons. 

**To Reproduce**
I just ran though these steps <https://github.com/ankitpokhrel/tus-php#docker> which are basically clone repo and execute `bin/docker.sh`. That all worked fine and the example site showed up when going to http://0.0.0.0:8080/ but an error occurred during the upload process. I tried in both the latest version of firefox and chrome.

**Screenshots**
![Screen Shot 2020-06-27 at 7 29 12 pm](https://user-images.githubusercontent.com/346340/85919197-9dd3a380-b8ac-11ea-9341-e03d814075fa.png)

**Comment by ankitpokhrel** [comment]
@callumgare I see CORs related error. Can you also share Response headers please? Does it contain `Access-Control-*` headers?

**Comment by callumgare** [comment]
I feel like I’m going a bit crazy. I had messed around for half an hour before hand trying different browsers, trying a bunch of different things to get it working and no cigar. After reading your message I tried it again to get the headers to send to you and it failed a couple times and then it started working and it has seem to continue working so I guess I can close this ticket now. No idea what happened but thanks for your support :)

![Screen Shot 2020-06-27 at 9 37 01 pm](https://user-images.githubusercontent.com/346340/85921796-26a90a00-b8c2-11ea-8d7b-1b9d55bbfa82.png)
